docs/assets/js/live-versions.js

```diff
@@ -51,8 +51,9 @@
             }
 
             const extraClass = (extra === "highlighted") ? "highlight" : "";
+            const returnText = (extra === "raw") ? `${cache(name, fetchedVersion)}` : `<span class="fetched-version ${type} ${extraClass}">${cache(name, fetchedVersion)}</span>`;
 
-            return `<span class="fetched-version ${type} ${extraClass}">${cache(name, fetchedVersion)}</span>`;
+            return returnText;
```

Comment:
I'd rather use an if statement here instead of a ternary
```suggestion
            const version = cache(name, fetchedVersion);
            if (extra === "raw") {
                return version;
            } else {
                return `<span class="fetched-version ${type} ${extraClass}">${version}</span>`;
            }
```

Reply:
(Added manually)